MONOREPO_ROADMAP.md

```diff
@@ -0,0 +1,383 @@
+- Utilities for evaluating [Datasources](#datasources)
+
+###### `@evidence-dev/sdk/plugins/app-shells`
+- Exposes functions for setting up the [Application Shell](#application-shell) and copying the [Project](#evidence-project) into it
+
+###### `@evidence-dev/sdk/usql`
+- Library code for interacting with USQL
+- `Query`, `Metric`, etc.
+
+###### `@evidence-dev/sdk/logging`
+- Provide functions for error, warn, log, and debug
+- Provide hooks to integrate logs into Evidence UIs (e.g. a log panel in VS Code)
+  - These hooks should have a standard interface for the contents of the logs
+- Wrap / re-use an existing, tested logging framework to minimize first-party code
+  - e.g. `winston` or `pino` should be used under the hood
+
+
+#### `@evidence-dev/lang-svelte`
+
+Responsible for providing a smooth interface between the [Evidence SDK](#evidence-devsdk) and Svelte. 
+
+This functionality was previously provided by `@evidence-dev/component-utilities`, `@evidence-dev/preprocessor` and `@evidence-dev/sdk/utils/svelte`
+
+##### Does:
+- Provide a preprocessor to transform Evidence Markdown files to Svelte files
+  - Extracts & Bootstraps Queries
+  - Injects [Component Plugins](#component-plugin)
+  - Initializes [USQL](#evidence-devusql-duckdb)
+- Provide an interface to ease the use of [sdk](#evidence-devsdk) functionality in Svelte (e.g. using Svelte's context or lifecycle functions)
+  - Includes making SSR in an [Application Shell (plugin)](#application-shell-plugin) or [Evidence App](#evidence-application) trivial
+
+##### Does Not:
+- Copy files from an [Evidence Project](#evidence-project) to an [Application Shell](#application-shell)
+
+### UI Packages
+
+#### `@evidence-dev/echarts`
+
+> [!WARNING]
+> This package would benefit from more definition and scoping.
+> It is not clear where the "boundary" is.  
+> For example, will it infer the Y column of a chart if one is not provided?
+
+> [!NOTE]
+> Some thought should be given to a similar pattern with mapping logic
+
+Responsible for encapsulating Evidence's echarts configurations in a re-usable package. This package is extracted to enable Evidence charts to be rendered in NodeJS applications.
+
+This functionality was previously provided by `@evidence-dev/core-components` and `@evidence-dev/component-utilities`
+
+##### Does:
+
+- Abstract echarts interactions away from Svelte (e.g. makes [`Chart.svelte`](./packages/ui/core-components/src/lib/unsorted/viz/core/_Chart.svelte))
+- Apply echarts theme + styling (incl. user overrides)
+
+##### Does Not:
+- Depend on Svelte, React or any other framework
+- Validate data (e.g. column existence)
+- Summarize data
+
+
+
+#### `@evidence-dev/ui-svelte`
+
+Responsible for providing Visualization and UI Components for building
+[Evidence Apps](#evidence-application) and [Evidence Projects](#evidence-project)
+
+This functionality was previously provided by `@evidence-dev/core-components` and `@evidence-dev/component-utilities`
+
```

Comment:
So, if you are using the sdk, you are not able to use these components, as they are svelte.